### ISSUE TITLE
Does it work in Oreo (Android 8.0.0)? 

### ISSUE THREAD

**Issue by Racroix** [mauron85/cordova-plugin-background-geolocation#396]
I am not good at English and I would like you to understand this please.

I think the backgroundgeolocation in Oreo(Android 8.0.0) does not work.

## Your Environment

* Plugin version:  plugin name="cordova-plugin-mauron85-background-geolocation" spec="^2.3.5"
* Platform: Android
* OS version: Oreo 8.0.0
* Device manufacturer and model: samsung, galaxy s9 plus 256G
* Cordova version (`cordova -v`): 8.0.0
* Cordova platform version (`cordova platform ls`): android 6.3.0
* Plugin configuration options: 
      let config = {
        desiredAccuracy: 0,
        stationaryRadius: 20,
        distanceFilter: 10,
        startForeground: false,
        debug: false,
        interval: 2000
      };

* Link to your project: none

## Context

It does not bring latitude and longitude.
It works on another phone. (samsung galaxy s6 edge, samsung galaxy note 4).

## Expected Behavior

Output latitude and longitude with push notifications in background mode.

## Actual Behavior

Current position output.
S6 edge =>
Results = title: xx.xxxxxx, text: xxx.xxxxxx
Note 4 =>
Results =  title: xx.xxxxxx, text: xxx.xxxxxx
S9 plus =>
Result = 0, 0
## Steps to Reproduce

my sample code... 

getLocation() {
if (!this.isRunning) {
      let config = {
        desiredAccuracy: 0,
        stationaryRadius: 20,
        distanceFilter: 10,
        startForeground: false,
        debug: false,
        interval: 2000
      };
      this.backgroundGeolocation.configure(config).subscribe((location: BackgroundGeolocationResponse) => {
        this.zone.run(() => {
          this.lat = location.latitude;
          this.lng = location.longitude;
        });

        if (this.platform.is('ios')) {
          this.backgroundGeolocation.finish();
        }

        this.backgroundGeolocation.stop();

      });

      this.backgroundGeolocation.start();

      let options = {
        frequency: 3000,
        enableHighAccuracy: true
      };

      this.watch = this.geolocation.watchPosition(options).filter((p: any) => p.code === undefined).subscribe((position: Geoposition) => {
        // Run update inside of Angular's zone
        this.zone.run(() => {
          this.lat = position.coords.latitude;
          this.lng = position.coords.longitude;
        });
      });
    }
    this.isRunning = true;
}

checkNotice(){
  let notice = {
      id: idNotification,
      title: this.lat,
      text:  this.lng
    };
    this.localNotifications.schedule(notice);
}

## Context

I want to get latitude and longitude results when in background mode on the S9 plus.

**Comment by mauron85** [comment]
What is the plugin version you're using?

**Comment by Racroix** [comment]
I use version 2.3.5.

**Comment by ftunholm** [comment]
I have done some tests today on galaxy S8 with Android 8.0.0. My plugin version is 2.3.5. Config as follows:
      desiredAccuracy: 0,
      stationaryRadius: 10,
      distanceFilter: 20,
      interval: 30000,
      debug: true,
      stopOnTerminate: false,
      startOnBoot: true,
      startForeground: false,
      pauseLocationUpdates: false,
      syncThreshold: 5,
      maxLocations: 20,
      stopOnStillActivity: false 

I have been trying my app out on OnePlusOne with Android 6.0.1 too, there it works flawless in foreground as well as in background.

On my S8 on the other hand I can't really say what is happening. It seems to send some location updated while in foreground. But it seems is fails to register that i'm in motion while in background. I took a walk for one hour and it did not fire once. 

I have changed my config a bit, ill try it again.
